src/setupProxy.js

```diff
@@ -0,0 +1,10 @@
+const { createProxyMiddleware } = require('http-proxy-middleware')
+module.exports = function(app) {
+  app.use(
+    '/api',
+    createProxyMiddleware({
+      target: `${process.env.REACT_APP_OJWANG_API_HTTP_SCHEME}://${process.env.REACT_APP_OJWANG_API_BASE_URL}`,
+      changeOrigin: true,
```

Comment:
We definitely don't have a CORS issue in prod, so this should not be checked in. I propose you ship this change without this proxy

Reply:
It's not read in production. I should have installed the http-proxy-middleware as a dev dependency. My bad.